lab-andrew/server.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
```

Comment:
This is a great start, I really like your bad request error message :D There were a few things which can be improved, please see my comments below.